### PR TITLE
[FIX](array )fix array<null> literal in fe

### PR DESCRIPTION
**Pull request by amorynan**
## Proposed changes

Issue Number: close #xxx
fix array<null> literal in fe const fold logic 

## Further comments

If this is a relatively large or complex change, kick off the discussion at [dev@doris.apache.org](mailto:dev@doris.apache.org) by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: fe/fe-core/src/main/java/org/apache/doris/analysis/NullLiteral.java [hunk 1]
```diff
@@ -101,6 +101,11 @@ public String getStringValue() {
         return "NULL";
     }
 
+    @Override
+    public String getStringValueInFe() {
+        return null;
```
Comment (morrySnow) [inline fe/fe-core/src/main/java/org/apache/doris/analysis/NullLiteral.java:106]: so all NullLiteral will return lower case null?

Comment (morrySnow) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/NullLiteral.java:106]: btw, u should also add ut to ensure `getStringValueInFe` return right string

Comment (amorynan) [reply fe/fe-core/src/main/java/org/apache/doris/analysis/NullLiteral.java:106]: no, origin logic just data.add(null) , here just keep origin logic

**Comment by amorynan** [comment]
run buildall

**Review by morrySnow — DISMISSED** [review]

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by amorynan** [comment]
run buildall

**Comment by amorynan** [comment]
run buildall

**Review by dataroaring — DISMISSED** [review]
LGTM

**Comment by dataroaring** [comment]
run buildall

**Review by dataroaring — APPROVED** [review]
LGTM

**Review by bobhan1 — APPROVED** [review]

**Review by dataroaring — APPROVED** [review]
LGTM

**Pull request merged**